Applications/Spire/Source/TimeAndSales/TimeAndSalesTableView.cpp

```diff
@@ -141,6 +118,30 @@ namespace {
     return properties;
   }
 
+  auto make_header_menu(TableView& table_view,
+      const std::vector<HeaderItemProperties>& properties) {
+    auto header = &table_view.get_header();
+    header->setContextMenuPolicy(Qt::CustomContextMenu);
+    auto menu = new ContextMenu(*header);
+    for(auto i = 0; i < std::ssize(properties); ++i) {
+      auto model = menu->add_check_box(header->get_items()->get(i).m_name);
```

Comment:
Avoid using the name "model" for nouns. Calling something a model is like calling something a variable, it doesn't add much information. Instead call the object by what it represents, in this case something like ```is_checked```.